about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+        <meta name="description" content="Global Streaming is live-streaming website that allow users to record and broadcast video simultaneously.">
+    
+        
+        <!-- Font Awesome kit Code-------- -->
+        <script
+          src="https://kit.fontawesome.com/b21ce73c3c.js"
+          crossorigin="anonymous"
+        ></script>
+    
+       <!-- Google fonts ------- -->
+       <link rel="preconnect" href="https://fonts.googleapis.com">
+       <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+       <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap" rel="stylesheet">
+       
+       
+    
+        <!-- local style links-->
+        <link rel="icon" href="images/logo.png" />
+         <link rel="stylesheet" href="./css/about.css">
+        
+        <title> Global Streaming </title>
+    </head>
+<body>
+    <header class="header">
+        <!-- top menu start-->
+        <div class="top-menu">
+            <ul class="top-menu-ul">
+                <li class="a"> <a href=""> <span class="fa fa-facebook"></span>
+                </a></li>
+                <li class="a"> <a href=""> <span class="fa fa-twitter"></span></a></li>
+                <li class="a"> <a href=""> English</a></li>
+                <li class="a"> <a href=""> Mypage</a></li>
+                <li class="a"> <a href=""> Logout</a></li>
+            </ul>
+        </div>
+         <!-- menu start-->
+        <nav class="main-menu">
+           <a href="index.html"> <img src="./images/logo.png" alt="logo"> <p> Global <span> Streaming</span> </p></a>
+           <span class="icon">
+            <span class="fa fa-bars"   aria-hidden="true"></span>
+            <span class="fa fa-times"   aria-hidden="true"></span>
+           </span>
+            <ul class="main-menu-ul">
+                
+                <li> <a href="about.html" class="active"> About </a></li>
+                <li><a href="index.html" > Home</a></li>
+                <li> <a href="#"> Live_Streaming </a></li>
+                <li class="disapper"> <a href="index.html#event-categories"> Event </a></li>
+                <li class="disapper"><a  href="#event">See recent</a></li>
+                <li class="disapper"> <a href="#">  Logout </a></li>
+            </ul>
+        </nav>
+    </header>
+<main>
+    <section class="descrption">
+    <h2>GLOBAL . STREAMING</h2>
+    <h2> BEST LIVE STREAMING WEBSITE -2022</h2>
```

Comment:
- [ ] The main title should be in `h1` tag and it should be same main title as the home page, I noticed that you changed the text, kindly fix this.